cmd/main.go

```diff
@@ -108,7 +107,7 @@ func main() {
 			klog.ErrorS(err, "failed to get version")
 			klog.FlushAndExit(klog.ExitFlushTimeout, 1)
 		}
-		fmt.Println(versionInfo)
+		klog.Info(versionInfo)
```

Comment:
This one we should probably put back in and `nolint`, it's normal to print version info entirely by itself, it shouldn't come with a timestamp like a log entry.